docs/world maintainer.md

```diff
@@ -0,0 +1,57 @@
+# World Maintainer
+
+A world maintainer is a person responsible for a world or part of a world in Archipelago.
+
+If a world author does not want to take on the responsibilities of a world maintainer, they can release their world as
+an unofficial [APWorld](/docs/apworld%20specification.md) or maintain their own fork instead.
+
+
+## Responsibilities
+
+Unless these are shared between multiple people, we expect the following from each world maintainer
+
+* Be on our Discord to get updates on problems with and suggestions for the world.
+* Decide if a feature (pull request) should be merged.
+* Review contents of such pull requests or organize peer reviews or post that you did not review the content.
+* Fix or point out issues when core changes break your code.
+* Use the watch function on GitHub, the #github-updates channel on Discord or check manually from time to time for new
+  pull requests. Core maintainers may also ping you if a pull request concerns your world,
+* Test (or have tested) the world on the main branch from time to time, especially during RC (release candidate) phases
+  of development.
+* Let us know of long unavailabilities.
+
+
+## Becoming a World Maintainer
+
+### Adding a World
+
+When we merge your world into the core Archipelago repository, you automatically become world maintainer unless you
+nominate someone else (i.e. there are multiple devs).
+
+### Getting Voted
+
+When a world is unmaintained, the [core maintainers](https://github.com/orgs/ArchipelagoMW/people)
+can vote for a new maintainer if there is a candidate. The majority of valid, duly votes has to be in favor.
+The time limit is 1 week, but can end early if the majority is reached earlier.
+Voting shall be conducted on Discord in #archipelago-dev.
+
+
+## Dropping out
+
+### Resigning
+
+A world maintainer can resign. If no new maintainer steps up and gets voted, the world becomes unmaintained.
+
+### Getting Voted out
+
+A world maintainer can be voted out by the [core maintainers](https://github.com/orgs/ArchipelagoMW/people),
+for example when they become unreachable. The majority of valid, duly votes has to be in favor.
+The time limit is 2 weeks, but can end early if the majority is reached earlier AND the world maintainer was pinged and
+made their case or was pinged and has been unreachable for more than 2 weeks already.
+Voting shall be conducted on Discord in #archipelago-dev.
```

Comment:
We can vote on Discord, but I would like the date of decision, voting members, and final result included in the commit message which moves a world to `worlds-disabled/`.

Reply:
It may never make it into the git history if we find a new maintainer while the old one drops out.
We can put "Commits that are a direct result of the voting shall include date, voting members and final result in the commit message" maybe?

Reply:
Yeah, that's fine.